### DISCUSSION TITLE
Custom video URL?

### DISCUSSION THREAD

**Discussion by 0x5c** [GriefPrevention/GriefPrevention#1263] [Q&A] [answered]
Hello!

I am currently setting up a server, and I've customised enough of GriefPrevention for the default video to be quite confusing to new users. The claim tool is a feather, the inspection tool is paper, there's blocks limits I want to explain, and I don't mind recording a custom explanation video.

However, by looking at the config and message files, there doesn't seem to be any setting about the URL. Is there even a way to change it?
There also does not seem to be any (open or closed) issues or PRs about this either.

**Comment by RoboMWM** [comment, marked as answer]
yes, just put in your own link and ignore the placeholder.

**Reply by Fituu** [reply]
Hi, I know this is a very old thing but I'm having the same problem, but the link doesn't become clickable on the claimbook and loses its color on the chat messages, is there any way to fix that?

**Reply by RoboMWM** [reply]
You can add color codes. However, the clickable-ness I think is likely a change from either spigot or paper's end on parsing messages. It used to be that URLs in sent messages were automatically parsed by the server (spigot etc.) to become clickable, but this has changed somewhere... I haven't identified when or what changed this behavior...

**Comment by Jikoo** [comment]
Change the message and exclude the replacement for the URL, put your own in its place.

**Comment by 0x5c** [comment]
Oh I didn't think that the URL would become clickable

That's what I get for diving too deep in NBT and JSON Text Components for other stuff

Thanks a lot!

**Reply by RoboMWM** [reply]
yea, I think CB/spigot does some detection of links in sendMessage and automatically applies the components to make them clickable.
